ci/gitlab/.gitlab-ci.yml

```diff
@@ -89,6 +90,28 @@ coverage:
     - aws s3 cp s3://ota-gitlab-ci/hereotaconnect_prod.zip $CI_PROJECT_DIR/credentials.zip
     - ./scripts/test.sh
 
+coverage-xenial:
+  variables:
+    TEST_BUILD_DIR: 'build-coverage-xenial'
+    TEST_CMAKE_BUILD_TYPE: 'Valgrind'
+    TEST_WITH_COVERAGE: '1'
+    TEST_WITH_P11: '1'
+    TEST_WITH_FAULT_INJECTION: '1'
```

Comment:
Let's try disabling the P11 and fault injection variables and then I think there's just a couple small things to finish up.